Lib/test/test_gettext.py

```diff
@@ -684,6 +684,13 @@ def test_plural_form_error_issue17898(self):
             # If this runs cleanly, the bug is fixed.
             t = gettext.GNUTranslations(fp)
 
+    def test_ignore_comments_in_headers_issue36239(self):
+        with open(MOFILE, 'wb') as fp:
+            fp.write(base64.decodebytes(GNU_MO_DATA_ISSUE_17898))
+        with open(MOFILE, 'rb') as fp:
+            t = gettext.GNUTranslations(fp)
+            self.assertEqual(t.info()["plural-forms"], "nplurals=2; plural=(n != 1);")
```

Comment:
Please add a comment explaining that the .mo file contains a comment. Something like:

Check that "#-#-#-#-#  messages.po (EdX Studio)  #-#-#-#-#" comment is ignored.

Reply:
good idea, done, thanks for proofreading Victor :)